.github/workflows/ci.yaml

```diff
@@ -25,4 +25,11 @@ jobs:
         node-version: ${{ matrix.node-version }}
     - run: npm ci
     - run: npm run build --if-present
-    - run: npm test
\ No newline at end of file
+    - run: npm run test-coverage
```

Comment:
Does this internally run test, so that unit tests, prettier, and the linter run?

Reply:
Yep. It since coverage takes a little longer I didn't know if we'd want to run it locally every time so I added the new action test-coverage. I figure we can make it always run coverage, if necessary, but started with this path.